modin/engines/base/io/text/csv_dispatcher.py

```diff
@@ -191,8 +201,26 @@ def _read_csv_check_support(
         if read_csv_kwargs.get("chunksize") is not None:
             return False
 
-        skiprows = read_csv_kwargs.get("skiprows")
-        if skiprows is not None and not isinstance(skiprows, int):
+        dummy_df = next(
```

Comment:
Why use `next`?

Reply:
Using iteration with `chunksize = 1` allows to reduce the time of getting columns number (from 0.8 sec to 0.2 sec on my machine for 5000x5000 asv dataset), looks like if we use `nrows=0` some buffering is present.

Reply:
It's not obvious, please add a comment to the code.

Reply:
Ok, comment is added.